verifier/trustpolicy/oci.go

```diff
@@ -53,41 +53,31 @@ type OCITrustPolicy struct {
 }
 
 // Document represents a trustPolicy.json document
+//
 // Deprecated: Document exists for historical compatibility and should not be used.
 // To create OCI Document, use OCIDocument.
 type Document = OCIDocument
 
 // TrustPolicy represents a policy statement in the policy document
+//
 // Deprecated: TrustPolicy exists for historical compatibility and should not be used.
 // To create OCI TrustPolicy, use OCITrustPolicy.
```

Comment:
Should we remove the comment?

Reply:
Tracking the discussion above: https://github.com/notaryproject/notation-go/pull/437#discussion_r1716264902